experimental/overhead_matching/swag/evaluation/evaluate_swag.py

```diff
@@ -156,24 +207,34 @@ def evaluate_model_on_paths(
         print("starting iter over paths")
         for i, path in enumerate(tqdm.tqdm(paths)):
             path_similarity_values = all_similarity[path]
-            motion_deltas = get_motion_deltas_from_path(vigor_dataset, path).to(device)
-            gt_initial_position_lat_lon = vigor_dataset._panorama_metadata.loc[path[0]]
-            gt_initial_position_lat_lon = torch.tensor(
-                (gt_initial_position_lat_lon['lat'], gt_initial_position_lat_lon['lon']), device=device)
-            generator.manual_seed(seed * i)
-            initial_particle_state = sa.initialize_wag_particles(
-                gt_initial_position_lat_lon, wag_config, generator).to(device)
-            particle_history = run_inference_on_path(sat_patch_kdtree,
-                                                     initial_particle_state,
-                                                     motion_deltas,
-                                                     path_similarity_values,
-                                                     wag_config,
-                                                     generator)
-            # save particle history
+            generator_seed = seed * i
```

Comment:
Is the initial seed saved somewhere? Consider saving the exact seed next to the evaluation path.

Reply:
In a different PR.